config/flags.go

```diff
@@ -369,6 +369,8 @@ func addNodeFlags(fs *pflag.FlagSet) {
 	fs.Float64(TracingSampleRateKey, 0.1, "The fraction of traces to sample. If >= 1, always sample. If <= 0, never sample")
 	fs.StringToString(TracingHeadersKey, map[string]string{}, "The headers to provide the trace indexer")
 	// TODO add flag to take in headers to send from exporter
+
+	fs.Bool(WriteRuntimeStateEnabledKey, false, "Whether to write runtime state (including PID, API URI, and boostrap address) to [base-data-dir]/runtime.json. If false (the default), runtime state will not be written.")
```

Comment:
This seems pretty different than the existing way we specify files.

Typically we specify a default location (which lives inside the `--data-dir`) [here](https://github.com/ava-labs/avalanchego/blob/master/config/flags.go#L35).

We then access the actual path using [GetExpandedArg](https://github.com/ava-labs/avalanchego/blob/master/config/flags.go#L405).

We actually (fairly painfully) broke the `--build-dir` flag in favor of the `--plugin-dir` with a default value living in the `--data-dir` so that we could be uniform here... So unless we have a _really_ good reason to deviate here... I feel like we should treat this the same.

Reply:
For all of our paths, we have a default location (which lives in the `--data-dir`). Additionally, the user can override this value to be anywhere they like. `GetExpandedArg` is mainly used as a way to evaluate the location of the file/dir honoring any changes to `--data-dir`.